src/Temporalio/Bridge/OptionsExtensions.cs

```diff
@@ -452,20 +469,14 @@ internal static class OptionsExtensions
                     throw new ArgumentException("Unable to get assembly manifest ID for build ID");
                 buildId = entryAssembly.ManifestModule.ModuleVersionId.ToString();
             }
-            var nonDetWorkflowFailForTypes = options.Workflows.Where(
-                w => w.FailureExceptionTypes?.Any(
-                    t => t.IsAssignableFrom(typeof(WorkflowNondeterminismException))) ?? false).
-                Select(w => w.Name).ToArray();
```

Comment:
This was unused

Reply:
Thanks! Wonder why there isn't an unused-local-var check in .NET, maybe I have it turned off